runtime/lua/vim/lsp/rpc.lua

```diff
@@ -421,6 +421,10 @@ function Client:handle_body(body)
   end
   log_debug('rpc.receive', decoded)
 
+  if body == nil or decoded == nil then
+    return
+  end
```

Comment:
This is not a proper fix. First, when `body = nil`, `vim.json.decode(nil)` would have thrown errors so it were already handled by `on_error`. Second, if the JSON itself is "null" (for somehow), something wrong happened in the JSON-RPC communication; possibly the bug of the specific LSP server. This should not be simply silenced, but fixed somewhere appropriately after finding the "root cause".